warehouse/internal/repo/schema.go

```diff
@@ -199,7 +363,8 @@ func (sh *WHSchema) GetNamespace(ctx context.Context, sourceID, destID string) (
 	query := `SELECT namespace FROM ` + whSchemaTableName + `
 		WHERE
 			source_id = $1 AND
-			destination_id = $2
+			destination_id = $2 AND
```

Comment:
Why are we adding empty table name here? Even if the SQL query picks up a table level entry, it will have the correct namespace, rt ?

Reply:
Yes. Just to be safe we are only looking for old entries which will have empty table name. As part of cleanup we will be removing those.
https://linear.app/rudderstack/issue/WAR-859/cleanup-for-table-level-schema-stop-populating-schema-and-no